packages/itwin/map-layers/src/test/MapManagerSettings.test.tsx

```diff
@@ -413,7 +413,7 @@ describe("MapManagerSettings", () => {
 
   it("exaggeration", () => {
     const component = mountComponent();
-    const numericInputs = component.find(NumberInput);
+    const exaggerationInput = component.find(Input).last();;
```

Comment:
I am very hesitant to rely on finding an array of inputs and using the 'last' input found, it's not foolproof. I would recommend adding a data-testid attribute to the Input component you are looking for. See [docs](https://testing-library.com/docs/queries/bytestid/) ref. I've also used data-testid in a previous PR for map-layers, so you can also find code examples within here